### DISCUSSION TITLE
[Request LCX] CHANGEDETECTION.IO with the playwright content fetcher

### DISCUSSION THREAD

**Discussion by Koliera** [tteck/Proxmox#541] [Ideas]
Would love this CHANGEDETECTION.IO (and would love to move over from Higgin). The playwright content fetcher (https://github.com/dgtlmoon/changedetection.io/wiki/Playwright-content-fetcher) allows to use the graphical interface to select the specific part of the webpage to track. I think this tool is a great addition to the scrips offered,
An example of this can be seen there - https://www.youtube.com/watch?v=0-BhkZJOvtg

**Comment by Koliera** [comment]
Bump, any chance someone could take a look?

**Comment by tteck** [comment]
This doesn't sound good
> There seems to be some memory leak in playwright https://github.com/microsoft/playwright/issues/6319 , as yet there does not seem to be a solution

**Reply by Koliera** [reply]
It seems like the issue is not always present. Besides - isn't the case that a memory leak would only affect the lcx in question?

**Comment by protonaut** [comment]
I'm also looking forward to see the playwright content fetcher involved in the proxmox install script.
